docs/reference-guides/core-blocks.md

```diff
@@ -987,6 +987,28 @@ Display the description of categories, tags and custom taxonomies when viewing a
 -	**Supports:** align (full, wide), color (background, link, text), interactivity (clientNavigation), spacing (margin, padding), typography (fontSize, lineHeight), ~~html~~
 -	**Attributes:** textAlign
 
+## Term Template
```

Comment:
Should we have consistent plurals? Either `Terms Template` or `Term Query`?

Reply:
> Actually maybe this way makes sense since the Query is for all terms but the template is for one.

This was the main reason I went with "terms" for the container block and "term" for the inner blocks, as the container is for all terms, but the inner blocks are for each individual term. But every time I've named one of the blocks I think about this 😅 I think this is pretty easy to change later, as the block is still experimental.